### DISCUSSION TITLE
Action ignores included path

### DISCUSSION THREAD

**Discussion by rbellamy** [community/community#26510] [Actions]
<p>The <a href="https://github.com/terradatum/base-github-runner/blob/master/.github/workflows/main.yml#L8" rel="nofollow noopener">Workflow and path</a>. The <a href="https://github.com/terradatum/base-github-runner/commit/e7e83fef78a7fb5912f263a781343235daa71746%20" rel="nofollow noopener">commit</a>.</p>
<p>Clearly the changed files should have triggered the workflow, but nothing happened:</p>
<p><a href="https://github.com/terradatum/base-github-runner/actions" rel="nofollow noopener">https://github.com/terradatum/base-github-runner/actions</a></p>
<p>Is there a known issue, or am I missing something? Is there a way to check an audit log of events and their listeners?</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/rbellamy">@rbellamy</a> ,</p>
<p>I notice you are using the pattern ’ <strong>ubuntu1?04.json</strong>’ on the push event in your workflow. I think you confuse the <strong>*</strong> character with <strong>?</strong> character in the pattern for paths filter.<br>
The <strong>?</strong> character matches zero or one of the preceding character, so the pattern ’ <strong>ubuntu1?04.json</strong>’ only can match the following paths:</p>
<ul>
<li><strong>ubuntu104.json</strong></li>
<li><strong>ubuntu04.json</strong></li>
</ul>
<p>If you want the pattern can match below paths in your repository, you should use the <strong><em></em></strong><em> character (’ <strong>ubuntu1*04.json</strong>’), instead of the <strong>?</strong> character. The <strong></strong></em> wildcard matches any character, but does not match slash (/).</p>
<ul>
<li><strong>ubuntu1604.json</strong></li>
<li><strong>ubuntu1804.json</strong></li>
</ul>
<p>More details, you can reference the docs about <a href="https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#patterns-to-match-file-paths" rel="nofollow noopener"><strong>patterns to match file paths</strong></a>.</p>

**Comment by rbellamy** [comment]
<p>And so it is. I read that documentation over and over, and never picked up on that - I just skipping past it and thinking knew what I was reading because I “know my globbing and wildcards.” THANK YOU.</p>
